examples/chart/teleport-cluster/values.schema.json

```
@@ -698,7 +698,19 @@
                 "create": {
                     "$id": "#properties/service/create",
                     "type": "boolean",
-                    "default": true 
+                    "default": true
+                }
+            }
+        },
+        "clusterRole": {
+            "$id": "#/properties/clusterRole",
```

Think this needs to be `rbac` instead?
```suggestion
        "rbac": {
            "$id": "#/properties/rbac",
```